packages/typography/README.md

```diff
@@ -10,10 +13,43 @@ yarn add @automattic/typography
 
 ## Usage
 
-Import the Sass file:
+Note that this package contains two sass files and there are use cases for `@import`ing either file.
+
+### Import the `variables.scss` file:
+
+`@import '~@automattic/typography/styles/variables';`
+
+Apply font variables as needed:
+
+```
+.my-text {
+    font-size: $font-body-small;
+}
+```
+
+### Import the `fonts.sccc` file:
 
 `@import '~@automattic/typography/styles/fonts';`
 
-Then apply the class name `wp-brand-font` to any elements that should display with Recoleta:
+Extend the .wp-brand-font selector in your SCSS:
+
+```
+.design__typography-branded {
+	@extend .wp-brand-font;
+	font-size: $font-title-medium;
+}
+```
+
+Or apply the class name `wp-brand-font` to any elements that should display with Recoleta:
 
 `<h1 className="wp-brand-font">Brand font heading</h1>`
+
+Note that the `fonts.scss` file imports the `variables.scss` file.
+
+<br>
+
+
+Please refer to the [Calypso Typography Docs](https://wpcalypso.wordpress.com/devdocs/typography) for more information.
+
+// Add usage about @import for both fonts.scss and variables.scss
```

Comment:
@autumnfjeld looks like these two `//` comments at the bottom can be removed now :)

Reply:
oh yes! thanks for catching!